engine/Build.bee.cs

```diff
@@ -156,62 +156,102 @@ static void DeployWindows()
+        foreach (var dep in dependencies_d)
         {
-            Backend.Current.AddAliasDependency("android", dep);
+            Backend.Current.AddAliasDependency("android_debug", dep);
+        }
+        foreach (var dep in dependencies_r)
+        {
+            Backend.Current.AddAliasDependency("android_release", dep);
         }
     }
 
     static void DeployMac()
     {
-        var libUIWidgets = SetupLibUIWidgets(UIWidgetsBuildTargetPlatform.mac, out var dependencies);
+        var libUIWidgets = SetupLibUIWidgets(UIWidgetsBuildTargetPlatform.mac, out var dependencies_d, out var dependencies_r);
 
         var nativePrograms = new List<NativeProgram>();
         nativePrograms.Add(libUIWidgets);
         var xcodeProject = new XCodeProjectFile(nativePrograms, new NPath("libUIWidgetsMac.xcodeproj/project.pbxproj"));
 
-        Backend.Current.AddAliasDependency("mac", new NPath("libUIWidgetsMac.xcodeproj/project.pbxproj"));
-        foreach (var dep in dependencies)
+        Backend.Current.AddAliasDependency("mac_debug", new NPath("libUIWidgetsMac.xcodeproj/project.pbxproj"));
+        foreach (var dep in dependencies_d)
         {
-            Backend.Current.AddAliasDependency("mac", dep);
+            Backend.Current.AddAliasDependency("mac_debug", dep);
+        }
+        foreach (var dep in dependencies_r)
+        {
+            Backend.Current.AddAliasDependency("mac_release", dep);
         }
     }
 
     //bee.exe ios
     static void DeployIOS()
     {
-        var libUIWidgets = SetupLibUIWidgets(UIWidgetsBuildTargetPlatform.ios, out var dependencies);
+        var libUIWidgets = SetupLibUIWidgets(UIWidgetsBuildTargetPlatform.ios, out var dependencies_d, out var dependencies_r);
         var nativePrograms = new List<NativeProgram>();
         nativePrograms.Add(libUIWidgets);
         var xcodeProject = new XCodeProjectFile(nativePrograms, new NPath("libUIWidgetsIOS.xcodeproj/project.pbxproj"));
 
-        Backend.Current.AddAliasDependency("ios",  new NPath("libUIWidgetsIOS.xcodeproj/project.pbxproj"));
-        foreach(var dep in dependencies) {
-            Backend.Current.AddAliasDependency("ios", dep);
+        Backend.Current.AddAliasDependency("ios_debug",  new NPath("libUIWidgetsIOS.xcodeproj/project.pbxproj"));
+        foreach(var dep in dependencies_d) {
+            Backend.Current.AddAliasDependency("ios_debug", dep);
+        }
+        foreach(var dep in dependencies_r) {
+            Backend.Current.AddAliasDependency("ios_release", dep);
         }
     }
 
     static void Main()
     {
-        flutterRoot = Environment.GetEnvironmentVariable("FLUTTER_ROOT");
+        flutterRoot = Environment.GetEnvironmentVariable("FLUTTER_ROOT_PATH");
         if (string.IsNullOrEmpty(flutterRoot))
         {
             flutterRoot = Environment.GetEnvironmentVariable("USERPROFILE") + "/engine/src";
         }
         skiaRoot = flutterRoot + "/third_party/skia";
 
+         try
```

Comment:
this file "Scripts/bitcode.conf" might not exist at all, is it ok to read from it directly here?